src/mantine-core/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -136,6 +136,9 @@ export const Autocomplete = forwardRef<HTMLInputElement, AutocompleteProps>(
     const { mergedStyles, rest } = useExtractedMargins({ others, style });
     const [dropdownOpened, _setDropdownOpened] = useState(initiallyOpened);
     const [hovered, setHovered] = useState(-1);
+    const [direction, setDirection] = useState<'column' | 'column-reverse'>('column');
```

Comment:
I'm not satisfied with that typing, how can I make it shareable - it comes from SelectDropdown component

Reply:
I think you can use `string` here or `React.CSSProperties['flexDirection']`